1.illumination_correction/config_files/config.yml

```diff
@@ -0,0 +1,18 @@
+channels: # Keep empty since we are using a Harmony v7 output
```

Comment:
I think it would be good to add a README in the `config_files` folder to describe why there are so many individual config files

Reply:
I will add a README now with the reasoning!